pkg/serverless/invocationlifecycle/trace_test.go

```diff
@@ -91,7 +91,7 @@ func TestEndExecutionSpanWithError(t *testing.T) {
 		tracePayload = payload
 	}
 
-	endExecutionSpan(mockProcessTrace, "test-request-id", endTime, isError)
+	endExecutionSpan(mockProcessTrace, "test-request-id", endTime, isError, "{}")
```

Comment:
can we add a test with an actual payload? + I think `meta["request"]` is untested

Reply:
added meta["function.request"] and meta["function.response"] test logic to `TestEndExecutionSpanWithNoError`